### ISSUE TITLE
Expose keywords in EclipseState

### ISSUE THREAD

**Issue by pgdr** [OPM/opm-parser#1089]
The following keywords should be exposed from EcliseState in one way or another in the attempt of removing exposure of Deck out of parser.

* NOGRAV
* THPRES (only to check if threshold pressures are enabled or not)
* EQUIL (ditto)
* ROCK

See discussion in OPM/ewoms#189

**Comment by joakim-hove** [comment]
@andlaus: 

**EQUIL:**

```C++
bool has_equil = eclState.getInitConfig().hasEquil( );
```

**THPRES:**
```C++
bool has_thpres = eclState.getSimulationConfig( ).hasThresholdPressure();
```

**ROCK:** (slightly uncertain on this one ...)
```C++
bool has_rock = eclState.getTableManager( ).hasTables("ROCK");
```

`NOGRAV` is not in any way internalized in `EclipseState`.

**Comment by andlaus** [comment]
yeah, I thought that most of this was somehow accessible. finding out which sub-object of EclipseState provides that information is non-trivial, don't you agree?

**Comment by joakim-hove** [comment]
> finding out which sub-object of EclipseState provides that information is non-trivial, don't you agree?

Well - I don't agree, but that is probably not very relevant.

**Comment by andlaus** [comment]
> Well - I don't agree, but that is probably not very relevant.

you probably don't agree because you know every obscure corner of this code. This is not the case for me anymore, so I have problems with this. Anyway, I agree that this has only limited relevance.

On a related matter: how to I get the rock parameters out of the RockTable object? (this is the only keyword which is presently not purely used as flag in the `EclProblem`)
